### PR TITLE
Add tests for pull-request-general-information

### PR DESCRIPTION
**Pull request by txsmith**

### CODE REVIEW

File: test/projects/pull-request/pull-request-general-information.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+              done();
+            }
+          }
+        });
+
+        elem.pullRequest = {};
+      });
+
+      test('has comments section', function() {
+        assert.isOk(get('pull-request-comments'));
```
Comment (eanker) [inline test/projects/pull-request/pull-request-general-information.html:68]: Are these sort of test usefull?
In other elements we do not have these tests.

File: test/projects/pull-request/pull-request-general-information.html [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        assert.isOk(get('marked-github-element'));
+      });
+
+      test('has user element', function() {
+        assert.isOk(get('a-user'));
+      });
+
+      test('has head branch element', function() {
+        var headElem = get('.head');
+        assert.isOk(headElem);
```
Comment (eanker) [inline test/projects/pull-request/pull-request-general-information.html]: What does this statement add?
If headElem is not truthy the next assert will fail.
(same for base branch)

Comment (txsmith) [inline test/projects/pull-request/pull-request-general-information.html]: You're right, this does not add much value to the test.

File: test/projects/pull-request/pull-request-general-information.html (hunk 1 above)
Comment (txsmith) [inline test/projects/pull-request/pull-request-general-information.html:68]: The tests ensure that the right information is on the page. Apart from this, there was not much to test.

Comment (eanker) [inline test/projects/pull-request/pull-request-general-information.html:68]: Ah, ok.
Just keep them, then.

**Comment by eanker** [comment]
lgtm

**Pull request merged**